docs/docs/development/api.md

```
@@ -487,6 +487,96 @@ https://myapp.example.com/callback?meetingID=test01&recordingmarks=true
 
 Another param is the `meetingEndedURL` create param. This create param is a callback to indicate the meeting has ended. This is a duplicate of the endCallbackUrl meta param. We have this separate as we want this param to stay on the server and not propagated to client and recordings. Can be used by scalelite to be notified right away when meeting ends. The meta callback url can be used to inform third parties.
 
+#### Learning Analytics Dashboard callback URL
+
+You can ask the BigBlueButton server to make a callback to your application when the meeting ends. Upon receiving the callback, your application can retrieve and save the Learning Analytics Dashboard data for long-term storage, analysis, or integration with your own learning management system.
+
+To specify the callback to BigBlueButton, pass a URL using the meta-parameter `meta_analytics-callback-url` on the `create` command. When the BigBlueButton server ends the meeting, it will check if `meta_analytics-callback-url` is set and, if so, make an HTTP GET request to the given URL.
+
+For example, to specify the callback URL as
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)